### PR TITLE
Bazel updates for gazebo4

### PR DESCRIPTION
**Pull request by JShep1**
Signed-off-by: John Shepherd <john@openrobotics.org>

### CODE REVIEW

**Comment by mahiuchun** [comment]
I see a link error due to `copyElement` being defined in both `ign_gazebo/src/ServerConfig.cc` and `ign_launch/plugins/gazebo_server/GazeboServer.cc`. Please use `static` or put them into anonymous namespace if they are meant to be local helper functions.

File: src/ign.cc [hunk 1]
```diff
@@ -357,6 +357,8 @@ extern "C" IGNITION_GAZEBO_VISIBLE int runServer(const char *_sdfString,
 //////////////////////////////////////////////////
 extern "C" IGNITION_GAZEBO_VISIBLE int runGui(const char *_guiConfig)
 {
+  Q_INIT_RESOURCE(gazebo);
```
Comment (mahiuchun) [inline src/ign.cc:360]: I had to remove this in order to avoid a dynamic link error.

**Review by mahiuchun — CHANGES_REQUESTED** [review]
It appears that this PR has not caught up with https://github.com/ignitionrobotics/ign-msgs/commit/e2709830f295061629eee65b2dcd876e80662556 yet so it does not build right now.

**Review by chapulina — CHANGES_REQUESTED** [review]
The build is broken

**Comment by chapulina** [comment]
@mjcarroll , I'll close this as it seems outdated

**Pull request closed without merging**